client/dashboard/task-list/tasks/payments/payfast.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Component, Fragment } from '@wordpress/element';
+import { Button } from '@wordpress/components';
+import interpolateComponents from 'interpolate-components';
+
+/**
+ * WooCommerce dependencies
+ */
+import { Link } from '@woocommerce/components';
+import { ADMIN_URL } from '@woocommerce/wc-admin-settings';
+
+/**
+ * Internal dependencies
+ */
+import { recordEvent } from 'lib/tracks';
+
+export default class PayFast extends Component {
+  constructor( props ) {
+    super( props );
+  }
+
+  continue = () => {
```

Comment:
After we save settings, we can use `markConfigured( 'payfast` )` here, create a success notice, and make sure we get redirected back to the task list after any selected gateways are configured.